brick/brick.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program determines if the brick can be inserted in the hole.")
+	var A, B float64
+	fmt.Println("Enter the hole dimensions (2 numbers):")
+	fmt.Scan(&A, &B)
+	var P, Q, R float64
+	fmt.Println("Enter the brick dimensions (3 numbers):")
+	fmt.Scan(&P, &Q, &R)
+	if A >= P && B >= Q || A >= P && B >= R || A >= Q && B >= P || A >= R && B >= P || A >= R && B >= Q || A >= Q && B >= R {
```

Comment:
Even if this condition is correct, it's very hard to understand it. What about trying to sort `P, Q, R` and `A, B`. Would it help to solve it any nicer?